### PR TITLE
add grpc type

### PR DESCRIPTION
**Pull request by isayme**

### CODE REVIEW

File: src/custom-types.json [hunk 1]
```diff
@@ -50,6 +50,12 @@
       "http://www.iana.org/assignments/media-types/application/geo+json"
     ]
   },
+  "application/grpc": {
+    "extensions": ["grpc"],
```
Comment (dougwilson) [inline src/custom-types.json]: There is no file extension listed in the source provided. Please provide a source that states the file extension for this type is defined to be .grpc

Comment (isayme) [reply src/custom-types.json]: sorry, my mistake. should add `proto` as extension for `application/grpc+proto`

File: src/custom-types.json [hunk 2]
```diff
@@ -50,6 +50,20 @@
...
     ]
   },
+  "application/grpc": {
+    "sources": [
+      "https://tools.ietf.org/html/draft-kumar-rtgwg-grpc-protocol-00",
+      "https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md"
+    ]
+  },
+  "application/grpc+proto": {
+    "extensions": ["proto"],
```
Comment (dougwilson) [inline src/custom-types.json]: I tried to look through the three docs, but I didn't see anything that says that `.proto` == `application/grpc+proto`. Can you let me know in which doc and where that is located?

File: src/custom-types.json [hunk 3]
```diff
@@ -50,6 +50,20 @@
       "http://www.iana.org/assignments/media-types/application/geo+json"
     ]
   },
+  "application/grpc": {
+    "sources": [
+      "https://tools.ietf.org/html/draft-kumar-rtgwg-grpc-protocol-00",
```
Comment (dougwilson) [inline src/custom-types.json]: This draft expired January 9, 2017. Once it is expired it is obsolete. Was the draft ever renewed or accepted? Please change the link to at least something not currently expired 👍

**Comment by dougwilson** [comment]
Closing due to no response. You're welcome to open a new PR with the comments above addressed.

**Pull request closed without merging**